geoapps/octree_creation/driver.py

```diff
@@ -67,21 +67,37 @@ def octree_from_params(params: OctreeParams):
             mesh_type="tree",
             depth_core=params.depth_core,
         )
+        minimum_level = max([1, treemesh.max_level - params.minimum_level + 1])
```

Comment:
This should be a private static method with signature: treemesh = OctreeDriver.minimum_level(treemesh, params)

Reply:
Done